### PR TITLE
support tls connection

### PR DESCRIPTION
**Pull request by hviana**

### CODE REVIEW

**Comment by hviana** [comment]
New feature finished.  No more commits

**Comment by hviana** [comment]
New feature finished. No more commits

**Comment by hviana** [comment]
New feature finished. No more commits.

**Comment by hviana** [comment]
New feature finished. No more commits

File: src/client.ts [hunk 1]
```diff
@@ -21,10 +21,20 @@ export class MongoClient {
     if (typeof options === "string") {
       options = parse(options);
     }
-    const conn = await Deno.connect({
+    var conn;
```
Comment (manyuanrong) [inline src/client.ts:24]: Do not use `any` at will. Don't use `var`

Comment (hviana) [reply src/client.ts]: Ok. Fixed.

**Comment by hviana** [comment]
I finished my commits. just used: any where it really is needed

**Comment by hviana** [comment]
I finished my commits. just used: any where it really is needed

**Comment by hviana** [comment]
the new feature is ok, and the new url parser is much more generic and powerful.

**Comment by hviana** [comment]
i finished my commits s

**Comment by hviana** [comment]
The new feature is ok, and the new url parser is much more generic and powerful. I finish my commits. You can Merge If you want.

**Comment by hviana** [comment]
The new feature is ok, and the new url parser is much more generic and powerful. I finish my commits. You can Merge If you want.

**Comment by hviana** [comment]
Also has minor fixes.

**Pull request merged**